hystrix-core/src/test/java/com/netflix/hystrix/util/LongAdderTest.java

```diff
@@ -0,0 +1,355 @@
+package com.netflix.hystrix.util;
```

Comment:
LongAdder tests look good. Were there any tests generated that give a non-zero output?

Reply:
unfortunately not.